.github/workflows/release.yml

```diff
@@ -8,5 +8,5 @@ on:
 
 jobs:
   call-terraform-release-pipeline:
-    uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-release.yml@v1.11.5
+    uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-release.yml@v1.12.2
```

Comment:
bump to v1.14.0 (has latest fixes)

Reply:
never mind, I merged it into main branch

Reply:
sure.